libexec/rtld-elf/rtld_c18n_policy.txt

```diff
@@ -55,6 +58,7 @@ trust
 	execvpe
 	execvP
 	unw_getcontext
+	unw_getcontext_unsealed
```

Comment:
Whoops, I should have mentioned that this should also belong to the first commit of the PR. And the commit message of this commit should be updated to remove the mention of moving symbols or adding missing symbols.